packages/cli/package.json

```diff
@@ -9,7 +9,10 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    ".": "./dist/index.js",
+    ".": {
+      "@vocab-private/monorepo": "./src/index.ts",
```

Comment:
I'm not sure why we started using `@vocab-private` instead of just having private packages with the `@vocab` scope instead.
I vaguely remember it being related to changesets (maybe?), and it was probably my fault, but it's likely unnecessary now. Happy to stick with it for now and change it later if you just want to ship this change.

Reply:
I thought it was intentional to not give the impression they were legitimate package. But yes would be happy to change in another PR.

Reply:
I think that's a moot point these days. `private: true` is good enough IMO - owning/managing multiple scopes (in NPM) seems like a worse option.